bitwarden/DOCS.md

```diff
@@ -97,6 +97,11 @@ especially if running with limited resources.
 
 To set the limit, you can use this setting: 10MB would be `10485760`.
 
+### Admin access
```

Comment:
This is listed in the installation steps, and should really be part of the BitWarden documentation itself.

Reply:
My bad :/